.github/workflows/create-deployment-spack.yml

```
@@ -67,4 +67,10 @@ jobs:
           git -C ${{ env.ROOT_VERSION_LOCATION }} clone https://github.com/ACCESS-NRI/spack-config.git --branch ${{ inputs.spack-config-version }}
           ln -s -r -v ${{ env.ROOT_VERSION_LOCATION }}/spack-config/v${{ steps.strip.outputs.version-dir }}/${{ vars.DEPLOYMENT_TARGET }}/* ${{ env.ROOT_VERSION_LOCATION }}/spack/etc/spack/
           mkdir ${{ env.ROOT_VERSION_LOCATION }}/release
+
+          # Create restricted folders
+          mkdir -p ${{ env.ROOT_VERSION_LOCATION }}/restricted/ukmo/release
+          mkdir -p ${{ env.ROOT_VERSION_LOCATION }}/restricted/ukmo/source_cache
+          # Only accessible to people in vk83_w, ki32_mosrs
+          setfacl -m "g:vk83_w:r-X,g:ki32_mosrs:r-X,g:vk83:---,other::---,d:g:vk83_w:r-X,d:g:ki32_mosrs:r-X,d:g:vk83:---,d:other::---" restricted
```

```suggestion
          setfacl -m "g:vk83_w:r-X,g:ki32_mosrs:r-X,g:vk83:---,other::---,d:g:vk83_w:r-X,d:g:ki32_mosrs:r-X,d:g:vk83:---,d:other::---" ${{ env.ROOT_VERSION_LOCATION }}/restricted
```
regardless this needs the fully qualified location